extension/icu/icu-timezone.cpp

```diff
@@ -278,6 +278,10 @@ struct ICUTimeZoneFunc : public ICUDateFunc {
 	}
 };
 
+timestamp_t ICUDateFunc::ICUFromLocal(icu::Calendar *calendar, timestamp_t local) {
```

Comment:
I will move this eventually but I'm messing with it in parallel and this will keep us from tripping over each other.

Reply:
I use `ICUFromNaiveTimestamp` introduced in [#5801](https://github.com/duckdb/duckdb/pull/5801) instead of `ICUFromLocalTimestamp`.